internal/nginx/modules/test/httpmatches.test.js

```diff
@@ -253,6 +253,14 @@ describe('headersMatch', () => {
       },
       expected: true,
     },
+    {
+      name: 'returns true if request has multiple values for a header name and one value matches ',
```

Comment:
I wonder why in this case we don't match across all header values. Is it our choice or is it specified by the API?